content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/tutorial-k8s.md

```diff
@@ -1,227 +1,388 @@
 ---
-date: '2021-09-21'
-title: "Deploying to a Local Kubernetes Cluster"
+date: '2021-11-14'
+title: "Deploying a Corda Cluster"
```

Comment:
If the title is "Deploying a Corda Cluster" - shouldn't the file be renamed as well ?